pylib/main.go

```diff
@@ -86,7 +123,9 @@ func cmdsToPy(cmds []dockerfile.Command) *C.PyObject {
 		C.Py_DecRef(pyEndLine)
 		C.Py_DecRef(pyFlags)
 		C.Py_DecRef(pyValue)
+		C.Py_DecRef(pyHeredocs)
 		C.Py_DecRef(ret)
+
 	}
```

Comment:
please review your PR and remove any unnecessary changes like this one (there are several)